tools/dbloadxml/colxml.cpp

```diff
@@ -46,7 +46,10 @@ int main(int argc, char** argv)
 #ifdef _MSC_VER
     //FIXME
 #else
+#pragma GCC diagnostic ignored "-Wunused-result"
     setuid( 0 ); // set effective ID to root; ignore return status
+    // Why should we raise privileges if we don't care?
```

Comment:
My guess is that failure there isn't always a real problem.  Failures later would tell the user whether it was a real problem or not.  But also this must be ancient code written under different specs.

Reply:
I agree with that guess. I will add extra message that notifies about the failure to simplify troubleshooting.